sensitive-data-protection.md

```diff
@@ -0,0 +1,172 @@
+# Sensitive Data Protection.
+
+This application stores sensitive personal data, and we have implemented robust measures to protect it. This document outlines how the data is encrypted, the encryption methods used, and the steps to decrypt it when necessary.
+
+
+## How to set up.
+
+After cloning the repository, you need to set up the frontend, and the backend to encrypt and decrypt the data.
+
+#### 1. You need to create a certificate (an RSA private/public key).**
+
+1. how to generate a certificate using openssl
+
+```bash
+# generate a private key with the 2048 bits (the recommended minimum)
+# you are going to be prompted to provide a password
+# Remember: it has a minimum of 4 characters
+openssl genrsa -aes256 -out private-key.pem 2048
+
+# generate a public key from the private key
+openssl rsa -in private-key.pem -pubout -out public-key.pem
+```
+
+_You can use alternative methods like `ssh-kegen` or `GnuPG` to generate the keys, among others._
+
+#### 2. For encrypting, you need to set up the frontend
+
+You are going to add your public key found at `public-key.pem` to the frontend. Because it is possible to use different keys eg: key rotation, 
```

Comment:
we should specify where in the frontend we add the public key and id... right?

Reply:
you can check the on line 31 the continuation documentation referring to the frontend how and where to add it.

I think we can not keep two same documentations. one on frontend and another one on this file. 

So here we say the high level and on the frontend/installation we go deep into how to set that up.